src/core/plugins/json-schema-2020-12-samples/fn/types/integer.js

```diff
@@ -27,12 +28,15 @@ const generateFormat = (schema) => {
 }
 const integerType = (schema) => {
```

Comment:
Can we insert an empty line here between `generateFormat` and `integerType`? Thanks!